pkg/api/api.go

```diff
@@ -96,6 +97,7 @@ func (a *API) RegisterRoutesWithPrefix(prefix string, handler http.Handler, auth
 
 //nolint:unparam
 func (a *API) newRoute(path string, handler http.Handler, isPrefix, auth, gzip bool, methods ...string) (route *mux.Route) {
+	handler = util.LogRequest(a.logger).Wrap(handler)
```

Comment:
Does that adds it for all requests including push requests ? We should avoid logging write requests, but also `/ready`, `/metrics` etc....